examples/rrt_panda.py

```diff
@@ -61,27 +61,27 @@
         # Search for a path
         planner = RRTPlanner(model, collision_model, options=options)
         path = planner.plan(q_start, q_end)
-        planner.visualize(viz, "panda_hand", show_tree=True)
+        planner.visualize(viz, "panda_hand", show_tree=True, show_path=False)
 
         if path:
             # Optionally shortcut the path
-            do_shortcutting = False
+            do_shortcutting = True
             if do_shortcutting:
                 path = shortcut_path(model, collision_model, path)
 
             discretized_path = discretize_joint_space_path(path, options.max_step_size)
 
-            if do_shortcutting:
```

Comment:
I think this if statement was here too because if shortcutting is false, then there should be no "shortened_path" visualization. As is now, if you have shortcutting disabled, the visualization will still be named "shortened_path" which could be misleading.

Additionally, if shortcutting is enabled, I also think you would not get to see the pre-shortcutting path, right? It would only plot the RRT and then the shortcut path. The idea is to have both paths available so you can toggle them on/off in Meshcat to compare.

Reply:
I renamed it to `planned_path` to avoid the misleading name.

Reply:
I'll put it back to how it was because we actually want the ability to visualize both `planned_path` (pre-shortcutting) and `shortcutted_path` (post-shortcutting) in the same run.